src/selectric.scss

```diff
@@ -108,6 +108,29 @@ $selectric-inner-height:    $selectric-height - ($selectric-border-width * 2) !d
     position: absolute;
     left: -100%;
   }
+
+  &.selectric-is-native {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+
+    select {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      height: 100%;
+      width: 100%;
+      border: none;
+      z-index: 1;
+      -webkit-box-sizing: border-box;
```

Comment:
Do you add vendor prefixes automatically while building? If yes I can simplify this a here...